src/components/table/OcTableFiles.vue

```diff
@@ -316,6 +336,14 @@ export default {
        */
       this.$emit("fileClick", resource)
     },
+
+    $_isResourceClickable(resourceId) {
+      const isDisabled = Array.isArray(this.disabled)
+        ? this.disabled.contains(resourceId)
+        : this.disabled === resourceId
+
+      return this.isResourceClickable && !isDisabled
```

Comment:
Would be good to have an early return on the prop. I.e. as a first check in the method: 
```
if (!this.isResourceClickable) {
   return false
}
```
And then 
```
return Array.isArray(this.disabled) ? this.disabled.contains(resourceId) : this.disabled === resourceId
```